app/themes/base.js

```diff
@@ -17,41 +15,30 @@ import {
   grey700,
   grey800,
   grey900,
-  fullBlack
+  darkBlack,
+  fullBlack,
+  white
 } from 'material-ui/styles/colors'
 
+// GK: theme structure following https://github.com/jxnblk/styled-system/blob/master/README.md
+
 export default {
-  breakpoints: {
-    mobile: '13.6em',
-    mobileWide: '30.4em',
-    tablet: '47.2em',
-    desktop: '64em',
-    desktopLarge: '97.6em'
-  },
-  em: '1em',
-  rem: '1rem',
-  fontSize1: '.875rem',
-  fontSize2: '1rem',
-  fontSize3: '1.25rem',
-  fontSize4: '1.5rem',
-  fontSize5: '2.25rem',
-  fontSize6: '3rem',
-  fontSize7: '5rem',
-  fontSize8: '6rem',
-  fontFamily: 'Roboto, sans-serif',
-  weightThin: 100,
-  weightLight: 300,
-  weightRegular: 400,
-  weightMedium: 500,
-  weightBold: 700,
-  weightBlack: 900,
-  colorPrimary1: teal500,
-  colorPrimary2: teal600,
-  colorPrimary3: teal700,
-  colorAccent1: deepPurple500,
-  colorAccent2: deepPurple600,
-  colorAccent3: deepPurple700,
-  colorGreys: [
+  breakpoints: [
+    32, 48, 64
+  ],
+  space: [
+    0, 6, 12, 18, 24
+  ],
+  fontSizes: [
+    12, 16, 18, 24, 36, 72
+  ],
+  colors: {
```

Comment:
i think these colors should still be "semantic" names, like `primary1` rather than `teal500`. i'd expect this change was only to nest the semantic names within the `colors` object, so `colorPrimary1` becomes `colors.primary1`, same for `accent1`, `text`, `alternativeText`, `canvas`, `border`, `shadow`.